Docs: RBAC GA (#49062)
grafana/grafana: docs/sources/enterprise/datasource_permissions.md

By default, data sources in an organization can be queried by any user in that organization. For example, a user with the `Viewer` role can issue any possible query to a data source, not just
queries that exist on dashboards they have access to.

> When permissions are enabled for a data source in an organization, the user who created the datasource will be able to edit the datasource and viewers can query by default.

Reviewer: ```suggestion
When permissions are enabled for a data source in an organization, the user who created the datasource will be able to edit the datasource and viewers can query by default.
```

> When permissions are enabled for a data source in an organization, the user who created the datasource will be able to edit the datasource and viewers can query by default.

Reviewer: Not sure what "...and viewers can query by default" means.

Is it something like "...the user who created the data source can edit the data source and the list of viewers who can query the data source"?

Reviewer 2: Trying to edit this to be more clear, something like 

"When permissions are enabled for a data source in an organization, the user who created the datasource can edit the datasource and in addition, viewers can query the datasource."

> When permissions are enabled for a data source in an organization, the user who created the datasource will be able to edit the datasource and viewers can query by default.

Reviewer 2: ```suggestion
When permissions are enabled for a data source in an organization, the user who created the datasource can edit the datasource and in addition, viewers can query the datasource.
```
